graphql/graphql.go

```diff
@@ -1187,6 +1134,7 @@ func (s *Service) Start(server *p2p.Server) error {
 	var err error
 	s.handler, err = NewHandler(s.backend)
 	if err != nil {
+		log.Warn("whaaa?", "err", err)
```

Comment:
Whaaa?

Reply:
Hah, obviously some debugging code I didn't mean to commit. 😊

Reply:
Force pushed 6bcd60f0f to remove this. :)